cypress/fixtures/smoke-tests/load-state-file-test-state_state.xml

```diff
@@ -0,0 +1 @@
pertyName>OWNER</ogc:PropertyName><ogc:Literal><![CDATA[Private]]></ogc:Literal></ogc:PropertyIsEqualTo></ogc:And></filter><actions><featureStyleAction><color>#ffffff</color><opacity>1</opacity><size>3</size><shape>Icon</shape><iconSrc>http://maps.google.com/mapfiles/kml/paddle/purple-stars.png</iconSrc><centerShape>Point</centerShape><showRotation>true</showRotation><rotationColumn>false</rotationColumn></featureStyleAction></actions></featureAction></featureActions><filters xmlns:ogc="http://www.opengis.net/ogc"><filter id="state#ghn7ebhnrjq2-fuz6f0gdzjvv" active="true" filterType="single" title="East Hydrants" description="" type="state#ghn7ebhnrjq2-depdqf9ttkbk-1904757301#features" match="AND"><ogc:And namehint="East Hydrants"><ogc:PropertyIsEqualTo><ogc:PropertyName>DIRECTION</ogc:PropertyName><ogc:Literal><![CDATA[E]]></ogc:Literal></ogc:PropertyIsEqualTo></ogc:And></filter></filters><queryAreas><queryArea id="state#ghn7ebhnrjq2-area_vt78stc771bb" title="Aurora Hydrant Include"><description></description><tags></tags><kml:Polygon xmlns:kml="http://www.opengis.net/kml/2.2"><kml:outerBoundaryIs><kml:LinearRing><kml:coordinates>-104.82165729127,39.61403121471881 -104.82165729127,39.77515550513715 -104.66969363246358,39.77515550513715 -104.66969363246358,39.61403121471881 -104.82165729127,39.61403121471881</kml:coordinates></kml:LinearRing></kml:outerBoundaryIs></kml:Polygon><layer>state#ghn7ebhnrjq2-depdqf9ttkbk-1904757301#features</layer></queryArea><queryArea id="state#ghn7ebhnrjq2-area_1k3s4yjthpe9" title="Aurora Police Include"><description></description><tags></tags><kml:Polygon xmlns:kml="http://www.opengis.net/kml/2.2"><kml:outerBoundaryIs><kml:LinearRing><kml:coordinates>-104.84757476992185,39.68512152332343 -104.84757476992185,39.75605766958 -104.80108188783922,39.75605766958 -104.80108188783922,39.68512152332343 -104.84757476992185,39.68512152332343</kml:coordinates></kml:LinearRing></kml:outerBoundaryIs></kml:Polygon><layer>state#ghn7ebhnrjq2-depdqf9ttkbk-3769575637#features</layer></queryArea></queryAreas><queryEntries><queryEntry layerId="state#ghn7ebhnrjq2-depdqf9ttkbk-1904757301#features" areaId="state#ghn7ebhnrjq2-area_vav6hrp4uoq3" filterId="state#ghn7ebhnrjq2-fuz6f0gdzjvv" includeArea="false" filterGroup="true"/><queryEntry layerId="state#ghn7ebhnrjq2-depdqf9ttkbk-1904757301#features" areaId="state#ghn7ebhnrjq2-area_vt78stc771bb" filterId="state#ghn7ebhnrjq2-fuz6f0gdzjvv" includeArea="true" filterGroup="true"/><queryEntry layerId="state#ghn7ebhnrjq2-depdqf9ttkbk-3769575637#features" areaId="state#ghn7ebhnrjq2-area_jj3494tspi0n" includeArea="false" filterGroup="true"/><queryEntry layerId="state#ghn7ebhnrjq2-depdqf9ttkbk-3769575637#features" areaId="state#ghn7ebhnrjq2-area_1k3s4yjthpe9" includeArea="true" filterGroup="true"/></queryEntries><time><interval>2019-01-07T00:00:00Z/2019-01-08T00:00:00Z</interval><current>2019-01-07T00:00:00Z/2019-01-08T00:00:00Z</current><advance>PT30M</advance><duration>day</duration></time></state>
```

Comment:
Depending on third-party servers for tests is not going to be consistent.

Reply:
https://github.com/ngageoint/opensphere/issues/398